config/shared/activities.js

```diff
@@ -61,6 +62,15 @@ function getLevel(type) {
   return schemaId && find(getOutlineLevels(schemaId), { type });
 }
 
+function getFromSameLevel(type) {
```

Comment:
Consider renaming this to getSiblingLevels in order to keep the consistent naming